Library/ViewModels/PledgePaymentMethodsViewModel.swift

```diff
@@ -325,6 +325,9 @@ public final class PledgePaymentMethodsViewModel: PledgePaymentMethodsViewModelT
           configuration.merchantDisplayName = Strings.general_accessibility_kickstarter()
           configuration.allowsDelayedPaymentMethods = true
 
+          // Enable Stripe Link
+          configuration.defaultBillingDetails.email = AppEnvironment.current.currentUserEmail
```

Comment:
Nit: If enabling stripe link is now this easy, is it a change we could/would want to flag guard after all?

Reply:
This is what happens if you don't pass in the e-mail - Stripe link is still there, i
<img width="533" alt="Screenshot 2024-06-10 at 4 37 49 PM" src="https://github.com/kickstarter/ios-oss/assets/146007185/ec862ee9-8056-429f-85b5-f553144b0d35">

This issue with the feature flag is that Stripe Link is effectively controlled by server-level settings, there's no way to enable/disable it purely on the client (without doing a ton of extra work on both front-end and back-end).